### ISSUE TITLE
False error report about unused component 'vue/no-unused-components'

### ISSUE THREAD

**Issue by paulgeisler** [vuejs/eslint-plugin-vue#1045]
**Tell us about your environment**	
* **ESLint version:** 6.8.0
* **eslint-plugin-vue version:** 6.1.2
* **Node version:** v10.15.3	

**Please show your full configuration:**

```js
module.exports = {
  extends: [
    'plugin:vue/recommended'
  ],
  parser: 'vue-eslint-parser',
  parserOptions: {
    parser: 'babel-eslint'
  }
}

```

**What did you do?**	

```vue
<script>
// ...
  import ModalLicenseInfo from '@sign_up/components/modal_license_info'

  export default {
    components: {
      ModalLicenseInfo
// ...
    },
    methods: {
      showLicenseInfo () {
        this.openModal(ModalLicenseInfo)
      }
//...
    }
```

**What did you expect to happen?**
No `error  The "ModalLicenseInfo" component has been registered but not used` error

**What actually happened?**
```
$ npx eslint --quiet --ext .js,.vue ./app/javascript/sign_up/

../app/javascript/sign_up/pages/step_account_details.vue
  20:7  error  The "ModalLicenseInfo" component has been registered but not used  vue/no-unused-components

✖ 1 problem (1 error, 0 warnings)
```

**Comment by ota-meshi** [comment]
Hello!

I think you can remove `ModalLicenseInfo` from `components` if you don't use components in the `<template>`.

```diff
    components: {
-      ModalLicenseInfo
// ...
    },
```

**Comment by paulgeisler** [comment]
You're right. Sorry for false report.
